src/initialization.js

```diff
@@ -25,16 +27,49 @@ var initialization = {
                 };
             } else {
                 isInitialized = true;
-                loadEventsAndPages();
+                loadWebXEventsAndPages();
             }
+
+            if (!window.optimizelyClientInstance && settings.useFullStack) {
+                common.userIdField = settings.userIdField;
+                var errorHandler = function() {};
+
+                if (customFlags && customFlags['OptimizelyFullStack.ErrorHandler']) {
+                    errorHandler = customFlags['OptimizelyFullStack.ErrorHandler'];
+                }
+
+                var instantiateFSClient = function() {
+                    window.optimizelyClientInstance = window.optimizelySdk.createInstance({
+                        datafile: window.optimizelyDatafile,
+                        errorHandler: {handleError: errorHandler}
+                    });
+
+                    window.optimizelyClientInstance.onReady().then(() => {
```

Comment:
annoying, but can you do:

```suggestion
                    window.optimizelyClientInstance.onReady().then(function() => {
```

Reply:
Some days my brain resists ES5 ways :) Done!